src/SDKs/DevTestLabs/DevTestLabs.Tests/Tests/BasicTests.cs

```diff
@@ -20,7 +20,7 @@ public void ListLabsTest()
             {
                 var client = GetDevTestLabsClient(context);
 
-                var labs = client.Lab.ListBySubscription().ToList();
+                var labs = client.Labs.ListBySubscription().ToList();
```

Comment:
This update was to get our sdk 'caught up' to what we already had in production.  We've started to work much more closely with the sdk team to make sure we do things differently moving forward to avoid changes like this.  But, to answer your question, these changes have been in production for a while now, we've just been latent in our publishing of the .net sdk that matches.